binder/environment.yml

```diff
@@ -55,11 +55,5 @@ dependencies:
   - xeus-python
   # extra conda stuff
   - jsonschema-with-format-nongpl
-  # jupyter-collaboration 1.2.0
-  - jupyter_ydoc >=1.0.1,<2.0.0
-  - ypy-websocket >=0.12.1,<0.13.0
```

Comment:
cc @davidbrochart looks like we could remove these dependencies and just depend on `jupyter-collaboration` directly?

Reply:
Indeed, that's how it should be.

Reply:
Binder resolved `ypy-websocket` to 0.8.2 - the previous range required a newer version (`>=0.12.1,<0.13.0`) - I guess this is ok because it works, but is this intended? 

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/bd8fccb7-0040-4eb9-be7c-b496a348e8bb)

Reply:
Still curious why `ypy-websocket` is installed?

Reply:
Is there a mamba command to check which package may depend on `ypy-websocket`?

Reply:
Wondering if this could be because of some of the pre-installed packages in the base Binder environment.

Reply:
> Is there a mamba command to check which package may depend on `ypy-websocket`?

I wanted to check, but the binder for this PR is not available anymore?

Reply:
We could use the one from `main` now: https://mybinder.org/v2/gh/jupyterlab/jupyterlab/main

(although it will have to build first)

Reply:
> Is there a mamba command to check which package may depend on `ypy-websocket`?

This one probably:

```
mamba repoquery whoneeds ypy-websocket
```

Reply:
I couldn't launch a binder, but at least from `main` no ypy package has been installed after installing jupyter-collaboration.